ivy/functional/backends/numpy/extensions/elementwise.py

```diff
@@ -82,3 +82,28 @@ def trapz(
 
 
 trapz.support_native_out = False
+
+
+def float_power(
+    x1: Union[np.ndarray, float, list, tuple],
+    x2: Union[np.ndarray, float, list, tuple],
+    /,
+    *,
+    out: Optional[np.ndarray] = None,
+) -> np.ndarray:
+    return np.asarray(
+        np.float_power(
+            x1,
+            x2,
+            where=True,
```

Comment:
done need specifying of arguments from where to subok as they are already set by default.